src/main/java/org/openelisglobal/analyzer/controller/AnalyzerErrorRestController.java

```diff
@@ -0,0 +1,311 @@
+package org.openelisglobal.analyzer.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.openelisglobal.analyzer.service.AnalyzerErrorService;
+import org.openelisglobal.analyzer.valueholder.AnalyzerError;
+import org.openelisglobal.common.exception.LIMSRuntimeException;
+import org.openelisglobal.common.rest.BaseRestController;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * REST Controller for Analyzer Error management
+ * 
+ * Task Reference: T095
+ * 
+ * Handles operations for error dashboard and reprocessing workflow: - List
+ * errors with filtering - Get error by ID - Acknowledge errors - Reprocess
+ * errors - Batch acknowledge
+ */
+@RestController
+@RequestMapping("/rest/analyzer")
+public class AnalyzerErrorRestController extends BaseRestController {
+
+    private static final Logger logger = LoggerFactory.getLogger(AnalyzerErrorRestController.class);
+
+    @Autowired
+    private AnalyzerErrorService analyzerErrorService;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    /**
+     * GET /rest/analyzer/errors
+     * 
+     * List analyzer errors with filtering and pagination
+     * 
+     * Query Parameters: - page, size, search, errorType, severity, status,
+     * analyzerId, startDate, endDate, sort
+     */
+    @GetMapping("/errors")
+    public ResponseEntity<Map<String, Object>> getErrors(@RequestParam(required = false) Integer page,
+            @RequestParam(required = false) Integer size, @RequestParam(required = false) String search,
```

Comment:
Not sure if it's still standard in OE, but ideally the strings at minimum should be validated as tightly as possible. Possibly work to schedule en masse later as I looked through a few rest controllers and didn't see it happening broadly